packages/app-project/src/screens/ClassifyPage/components/ClassifierWrapper/ClassifierWrapper.spec.js

```diff
@@ -104,82 +100,12 @@ describe('Component > ClassifierWrapper', function () {
     })
 
     describe('on classification complete', function () {
-      describe('with a new subject', function () {
-        before(function () {
-          const subject = {
-            id: '1',
-            already_seen: false,
-            favorite: false,
-            retired: false,
-            locations: [
-              { 'image/jpeg': 'thing.jpg' }
-            ]
-          }
-          wrapper.props().onCompleteClassification({}, subject)
-        })
-
-        after(function () {
-          yourStats.increment.resetHistory()
-          recents.add.resetHistory()
-        })
-
-        it('should increment stats', function () {
-          expect(yourStats.increment).to.have.been.calledOnce()
-        })
-
-        it('should add to recents', function () {
-          const recent = {
-            favorite: false,
-            subjectId: '1',
-            locations: [
-              { 'image/jpeg': 'thing.jpg' }
-            ]
-          }
-          expect(recents.add.withArgs(recent)).to.have.been.calledOnce()
-        })
-      })
-
-      describe('with a retired subject', function () {
+      describe('with user signed in and any subject', function () {
```

Comment:
These ClassifierWrapper tests are modified to "with user signed in and any subject" because ERAS counts all classifications as a +1 regardless of subject status as 'already seen' or 'retired'. Therefore, the behavior of user recents and user stats is the same in each scenario.

I also didn't include a test here for "increments stats on classification" because this spec file is written with implementation patterns and enzyme, and "increments stats on classification" requires looking at actual UI.